is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
+      if (elementToCompare === currentElement) {
+        isAllUnique = false;
+      }
+    });
+  });
```

Comment:
Is there a way to do this without a nested loop?

Reply:
perhaps a recursive solution?

Reply:
This solution improve space complexity.
```suggestion
  array.forEach((currentElement, currentIndex) => {
    if (array[currentIndex + 1, -1].includes(currentElement)) {
  isAllUnique = false;
}
    });
  });
```